code/script.js

```diff
@@ -0,0 +1,231 @@
+          document.getElementById(`comments-${reponame}`).innerHTML = ''
+        }
+      })
+    })
+    .catch(err => console.error(err))
+  })
+}
+
+/////////////////////////////////////////////////////////////////////////////
+////////////////////////       LANGUAGES DATA        ///////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+
+const languagesData = (username, reponame) => {
+
+  fetch(`https://api.github.com/repos/${username}/${reponame}/languages`, options)
+    .then(res => res.json())
+    .then(data => {
+    
+    // create an array with languages used by using the object keys. 
+    const arrayLanguages = (Object.keys(data))
+    arrayLanguages.forEach(item => {
+      document.getElementById(`languages-${reponame}`).innerHTML += `<span class="repo-languages--item">${item}</span>`    
+      console.log('item', arrayLanguages)
+    })    
+  }) 
+}
+
+////////////////////////////////////////////////////////////////////////////
+///////////////////////////      COMMITS DATA     /////////////////////////
+//////////////////////////////////////////////////////////////////////////
+const displayCommits = (commitsUrl, reponame) => {
+  fetch(commitsUrl, options)
+  .then(res => res.json())
+  .then(data => {
+    document.getElementById(`commits-${reponame}`).innerHTML = `
+    <p>
+    <span class="bold-text">Amount of commits:</span> ${data.length}
+    </p>`
+  })
+  .catch(err => console.error(err))
+}
+
+////////////////////////////////////////////////////////////////////////////
+//////////////////////////      COMMENTS DATA     /////////////////////////
+//////////////////////////////////////////////////////////////////////////
+const displayComments = (commentsUrl, reponame) => {
+  fetch(commentsUrl, options)
+  .then(res => res.json())
+  .then(data => {
+    if (data.length > 0) {
+
+      let commentsArray = []
+      
+      data.forEach(comment => {
+        if (comment.user.login !== username) {
+          document.getElementById(`comments-${reponame}`).innerHTML = `
+          <p>
+            <span class="bold-text">Reviewed by:</span> 
+            <a class="nav-link" href="${data[0].user.html_url}"> ${data[0].user.login}</a>
+          </p>
+          `         
+          document.getElementById(`comments-btn-${reponame}`).innerHTML = `
+            <button class="filled-button" id="click-${reponame}">comments 💬</button> 
+          `
+          commentsArray.push(comment.body)
+          document.getElementById(`click-${reponame}`).addEventListener('click', () => {
+            commentsArray.forEach(item => {
+              modalWrapper.style.display = 'block'
+        
+              document.getElementById('modal').innerHTML += `
+              <p><span class="bold-text modal-header">Comment from ${data[0].user.login}</span></p>
+              <p class="modal-text">${item}</p>
+              <br>
+              `
```

Comment:
It would be nice to add an else in this case to announce visitor that there is no comment. In term of user experience, I believe that it will be helpful to save them from questioning about where to find the comment for this project